src/meta.json

```diff
@@ -92,6 +92,9 @@
                 },
                 "staged": {
                     "type": "boolean"
+                },
+                "sourceOutputKey": {
+                    "type": "string"
```

Comment:
How about adding a description here, for example: "Output key associated with this input key"